toto/matchrule_validators.py

```diff
@@ -65,7 +67,7 @@ def _validate_generic_rule(keywords):
 
   toto.ssl_crypto.formats.PATH_SCHEMA.check_match(artifact)
 
-  if rule not in VALID_OPERATIONS:
+  if rule not in VALID_OPERATIONS and rule.upper() not in VALID_OPERATIONS:
     raise FormatError("{} is not a valid rule!".format(rule))
 
 def check_matchrule_syntax(keywords):
```

Comment:
The comment relates to `check_matchrule_syntax`'s docstring.
Name, Description, Returns, Raises != Purpose, Arguments, Exceptions, Side Effects, Return Value